app/src/main/res/values-vi/strings.xml

```diff
@@ -1,3 +1,251 @@
+    <string name="working_calendar">Lịch làm việc</string>
+    <string name="holiday_calendar">Lịch ngày nghỉ</string>
+    <string name="personal_request">YÊU CẦU CÁ NHÂN</string>
+    <string name="request_overtime">Yêu cầu làm ngoài giờ</string>
+    <string name="request_day_off">Yêu cầu xin nghỉ</string>
+    <string name="request_come_late_leave_early">Yêu cầu đi trễ, về sớm</string>
+    <string name="workspaces_data">DỮ LIỆU WORKSPACES</string>
+    <string name="workspaces">Dữ liệu Workspaces</string>
+    <string name="statistics_of_personal">Thống kê cá nhân</string>
+    <string name="manage_requests">QUẢN LÝ YÊU CẦU</string>
+    <string name="manage_request_overtime">Quản lý yêu cầu làm ngoài giờ</string>
+    <string name="manage_request_off">Quản lý yêu cầu xin nghĩ</string>
+    <string name="manage_request_leave">Quản lý yêu cầu đi trễ, về sớm</string>
+
+    <string name="create_request_ot">Tạo yêu cầu làm ngoài giờ</string>
+    <string name="create_request_leave">Tạo yêu cầu đi trễ, về sớm</string>
+    <string name="create_request_off">Tạo yêu cầu xin nghỉ</string>
+    <string name="navigation_drawer_open">Mở ngăn điều hướng</string>
+    <string name="navigation_drawer_close">Đóng ngăn điều hướng</string>
+    <string name="please_click_back_again_to_exit">Nhấn nút BACK lần nữa để thoát!</string>
+    <string name="email">Email</string>
+    <string name="password">Mật khẩu</string>
+    <string name="new_password">Mật khẩu mới</string>
+    <string name="current_password">Mật khẩu hiện tại</string>
+    <string name="confirm_password">Xác nhận mật khẩu</string>
+    <string name="forgot_password">Quên mật khẩu?</string>
+    <string name="log_in">Đăng nhập</string>
+    <string name="is_empty">Vui lòng nhập trường này !</string>
+    <string name="invalid_email_format">Email không đúng định dạng !</string>
+    <string name="least_6_characters">Vui lòng nhập ít nhất 6 ký tự !</string>
+    <string name="confirm_password_does_not_match">Xác nhận mật khẩu không khớp !</string>
+    <string name="request_leave">Yêu cầu đi trễ, về sớm</string>
+
+    <string name="employee_name">Tên nhân viên</string>
+    <string name="employee_code">Mã nhân viên</string>
+    <string name="project_name">Tên dự án</string>
+    <string name="position_name">Vị trí công việc</string>
+    <string name="branch">Chi nhánh</string>
+    <string name="group">Phòng ban</string>
+    <string name="contract_date"> Ngày bắt đầu làm việc chính thức</string>
+    <string name="start_probation_date">Ngày bắt đầu thử việc</string>
+    <string name="end_probation_date">Ngày kết thúc thử việc</string>
+    <string name="staff_type">Loại nhân viên</string>
+    <string name="position">Vị trí</string>
+    <string name="gender">Giới tính</string>
+    <string name="birthday">Ngày sinh</string>
+    <string name="off_have_salary_company_pay">Nghỉ có lương - Công ty trả</string>
+    <string name="number_day_off">Số ngày nghỉ</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.number_day_off` appears to be unused
 Unused resources make applications larger and slow down builds.